docker-compose.yml

```diff
@@ -50,9 +50,9 @@ services:
       - "5432:5432"
     environment:
       POSTGRES_PASSWORD: postgres
-      POSTGRES_DB: hsl_dev
+      POSTGRES_DB: members_api_db
     volumes:
-      - db_data:/var/lib/postgresql/data
+      - db_data_${NODE_ENV:-development}:/var/lib/postgresql/data
```

Comment:
Similarly, this value will read from `NODE_ENV` and fall back to `"development"` if no `NODE_ENV` value is available.